docs/narr/viewconfig.rst

```diff
@@ -285,6 +285,23 @@ Non-Predicate Arguments
   are just developing stock Pyramid applications. Pay no attention to the man
   behind the curtain.
 
+``accept``
+  This value represents a match query for one or more mimetypes in the
+  ``Accept`` HTTP request header.  If this value is specified, it must
+  be a specific media type, such as ``text/html``.  If the media type
+  matches the ``Accept`` header of the request, or if the ``Accept``
+  header isn't set at all in the request, this will match the current
+  view. If this does not match the ``Accept`` header of the request,
+  view matching continues.
```

Comment:
For view configuration, wouldn't view matching (unlike route matching) continue whether or not there is a match?

I see you've moved `accept` from under "Predicate Arguments" to under "Non-Predicate Arguments" &mdash; currently it seems the `accept` argument is handled as if it were both a non-predicate *and* a predicate argument? Both `best_match` and `__contains__` are used, with the call to `__contains__` seemingly unnecessary; and with the switch to `acceptable_offers`, `acceptable_offers` would be called twice. Am I reading it correctly, in that it seems `accept` shouldn't be a predicate at all for view configuration?

Reply:
Accept handling in pyramid is a predicate "with special extra support if you register more than one view for the same other-predicates but with different accept values".

It is used directly as a predicate in the following cases:

1) When used as a route predicate.
2) When only one view is registered and thus a MultiView is not created.
3) After the MultiView selects the view, it is checked again (an redundant check which we could optimize away).

Reply:
As far as why I moved this to the non-predicate arguments section, it was more than anything just to make it consistent with the current `add_view` docs. I'm probably going to move them both back to the predicate section before this is all said and done.